src/pages/licensees/index.jsx

```diff
@@ -11,23 +11,23 @@ export default function Licensees() {
 	const router = useRouter();
 
 	const actionColumn = {
-		field: 'actions',
-		headerName: 'Actions',
-		width: 150,
-		renderCell: (params) => {
-			return (
-				<IconButton
-					color='primary'
-					aria-label='go to details'
-					onClick={() => router.push(`/licensees/${params.row.id}`)}>
-					<MoreHorizIcon color='action' />
-				</IconButton>
-			);
-		},
+		// field: 'actions',
```

Comment:
do we need to keep this block?

Reply:
for now yes. I'd like to discuss with Scott before removing it completely.